infra/build/functions/build_lib.py

```diff
@@ -508,8 +508,8 @@ def get_build_body(steps,
   else:
     options = {}
 
-  if use_build_pool:
-    options['pool'] = {'name': OSS_FUZZ_BUILDPOOL_NAME}
+  # if use_build_pool:
+  #   options['pool'] = {'name': OSS_FUZZ_BUILDPOOL_NAME}
```

Comment:
nit: Should we remove it if it is not needed?

Reply:
Oh yeah this is test code.

Reply:
+1 let's remove.

Reply:
To be clear about what I mean: This code is needed, but I have to disable it to do builds on my local machine.